dbatools.psd1

```diff
@@ -350,9 +350,10 @@
         'Get-DbaRegServer',
         'Get-DbaRegServerGroup',
         'Get-DbaRegServerStore',
-        'Get-DbaRepDistributor',
-        'Get-DbaRepPublication',
-        'Get-DbaRepServer',
+        'Get-DbaReplDistributor',
+        'Get-DbaReplPublication',
+        'Get-DbaReplPublisher',
+        'Get-DbaReplServer',

```

Comment:
Think we are safe to move these to the section below, keep them all organized together.